planet request doesn't work anymore

I got these one in the latest CI: 

> FAILED tests/test_PlanetModel.py::TestPlanetModel::test_get_planet_items - planet.exceptions.BadQuery: {"general": [{"message": "PSScene3Band and PSScene4Band have been deprecated and are no longer available. To learn more, visit: https://developers.planet.com/docs/apis/data/psscene3-4band-deprecation/"}], "field": {}}

As explained in the issue, it seems they deprecated the PSScne completely now. I don't know if it's just a test update or if there is need for codebase update as well

--- a/sepal_ui/planetapi/planet_model.py
+++ b/sepal_ui/planetapi/planet_model.py
@@ -167,7 +167,7 @@ class PlanetModel(Model):
         )
 
         # PSScene3Band and PSScene4Band item type and assets are deprecated
-        # ince January 2023 so we are now only looking at PSScene
+        # since January 2023 so we are now only looking at PSScene
         item_types = ["PSScene"]
 
         async def _main():

--- a/sepal_ui/planetapi/planet_model.py
+++ b/sepal_ui/planetapi/planet_model.py
@@ -166,11 +166,9 @@ class PlanetModel(Model):
             ]
         )
 
-        # PSScene3Band and PSScene4Band item type and assets will be deprecated by January 2023
-        # But we'll keep them here because there are images tagged with these labels
-        # item types from https://developers.planet.com/docs/apis/data/items-assets/
-
-        item_types = ["PSScene", "PSScene3Band", "PSScene4Band"]
+        # PSScene3Band and PSScene4Band item type and assets are deprecated
+        # ince January 2023 so we are now only looking at PSScene
+        item_types = ["PSScene"]
 
         async def _main():
             """

